src/MCPClient/lib/clientScripts/emailFailReport.py

```diff
@@ -54,25 +50,21 @@ def get_emails_from_dashboard_users():
     return User.objects.filter(is_active=True).values_list('email', flat=True).exclude(email__in=['demo@example.com', ''])
 
 
-def send_email(subject, to, efrom, content, server):
-    logger.info('Sending report by email (to=%s, server=%s)', to, server)
-    to2 = ", ".join(to)
-    msg = MIMEMultipart('alternative')
-    msg['Subject'] = subject
-    msg['From'] = efrom
-    msg['To'] = to2
-
-    part1 = MIMEText(content, 'html')
-
-    msg.attach(part1)
-
+def send_email(subject, to, efrom, content):
     try:
-        server = smtplib.SMTP(server)
-        server.sendmail(efrom, to, msg.as_string())
-    except (smtplib.SMTPException):
+        logger.info('Sending email...')
+        return send_mail(
+            subject=subject,
+            message='Please see the attached HTML document',
+            html_message=content,
+            from_email=efrom,
+            recipient_list=to,
+        )
+    except (SMTPException):
```

Comment:
Is the reason we're keeping `SMTPException` and `smtplib` in here as well as using `django.core.mail` so that the interface and expectation for callers remains the same?

Reply:
I think it's because `django.core.mail` exposes those exceptions from the core `smtplib`. I guess other exceptions could arise if a different email backend is being used?

Reply:
Good point. It's not clear in their docs but it makes sense, e.g. I've checked that the SES backend would raise a `botocore.exceptions.ClientError` exception so I've added the except clause without exception names to catch them all. The only intention is to log the exception.

I've updated the commit.